ansible/roles/ooni-backend/tasks/cron.yml

```diff
@@ -0,0 +1,15 @@
+---
+- name: Template update bouncer script
+  template: >
+    src="update_bouncer.j2"
```

Comment:
Based on the comments below this should probably not be a template, but rather just be a file that is copied.